multichain.go

```diff
@@ -211,30 +213,28 @@ func (asset Asset) OriginChain() Chain {
 		return Filecoin
 	case FTM:
 		return Fantom
+	case GETH:
+		return Goerli
 	case GLMR:
 		return Moonbeam
 	case KAVA:
 		return Kava
 	case LUNA:
 		return Terra
-	case MATIC:
+	case MATIC, USDC_Polygon, USDT_Polygon:
 		return Polygon
 	case oETH:
 		return Optimism
 	case SOL:
 		return Solana
-	case UST:
-		return Terra
 	case ZEC:
 		return Zcash
 
 	case BADGER, BUSD, CRV, DAI, EURT, FTT, ibBTC, KNC, LINK, MIM, REN, ROOK,
 		SUSHI, UNI, USDC, USDT:
 		return Ethereum
 
-	case KETH:
-		return Kovan
-	case GETH:
+	case DAI_Goerli, REN_Goerli, USDC_Goerli, USDT_Goerli:
 		return Goerli
```

Comment:
Maybe this can be combined with `GETH` like what we do to `Polygon` and `Avalanche`